content/general/versions/v0.8/authentication.textile

```diff
@@ -127,23 +128,39 @@ ably.requestToken({ clientId: 'client@example.com' }, function(err, token) {
+"API keys":https://support.ably.io/solution/articles/3000030054-what-is-an-app-api-key, like "tokens":#tokens, have a set of capabilities assigned to them that specify which "operations":#capability-operations (such as subscribe or publish) can be performed on which channels. However, unlike tokens, API keys are long-lived, secret and typically not shared with un-trusted clients.
 
-API keys and their capabilities are "configured using the dashboard":https://support.ably.io/support/solutions/articles/3000030502-setting-up-and-managing-api-keys, they cannot be added or removed programmatically. Tokens on the other hand are designed to be shared with untrusted clients, are short-lived, and significantly, they are configured and issued programmatically using the Ably client libraries or directly from the "Ably REST API":/rest-api. See "selecting an authentication scheme":#selecting-auth to understand why token authentication, in most cases, is the preferred authentication scheme.
+API keys and their capabilities are "configured using the dashboard":https://support.ably.io/support/solutions/articles/3000030502-setting-up-and-managing-api-keys, they cannot be added or removed programmatically. Tokens on the other hand are designed to be shared with un-trusted clients, are short-lived, and significantly, they are configured and issued programmatically using the Ably client libraries or directly from the "Ably REST API":/rest-api. See "selecting an authentication scheme":#selecting-auth to understand why token authentication, in most cases, is the preferred authentication scheme.
 
 "Tokens":#tokens are issued from an existing "API key":https://support.ably.io/solution/articles/3000030054-what-is-an-app-api-key, and their capabilities can, at most, match the capabilities of the issuing API key. If an API key must be shared with a third party, then it is recommended that "the principle of least privilege":http://en.wikipedia.org/wiki/Principle_of_least_privilege is considered assigning only the capabilities needed by that third party. Thus, any Ably requests authenticated using that API key or tokens issued from that API key, will be restricted to the capabilities assigned to the key.
 
 Capabilities for tokens are determined as follows:
 
 * If no capability is specified in the token request, then the token will be given the full set of capabilities assigned to the issuing key, "see example":#capabilities-explained-all;
-* If a set of capabilities are requested, then the token will be assigned the intersection of the requested capability and the capability of the issuing key, "see example":#capabilities-explained-intersection;
+* If a set of capabilities are requested, then the token will be assigned the intersection of the requested capability and the capability of the issuing key, "see example":#capabilities-explained-intersection.
```

Comment:
Good point, will fix. (That wasn't actually a change I made directly -- I just copied over this whole section from current to v0.8, and in current that para already ended with a period for some reason)